docs/.vitepress/vitepress/components/vp-demo.vue

```diff
@@ -48,7 +57,23 @@ const props = defineProps({
     type: Object,
     required: true,
   },
+  rawSource: {
+    type: String,
+    required: true,
+  },
+  description: {
+    type: String,
+  },
+})
+
+const vm = getCurrentInstance()
+
+const { copy, copied, isSupported } = useClipboard({
+  source: ref(decodeURIComponent(props.rawSource)),
```

Comment:
set `read` to `false`

We don't need to read user's clipboard.

Reply:
https://vueuse.org/core/useClipboard/

Reply:
Done!